test_submissions/varcat/batch.json

```diff
@@ -0,0 +1,179 @@
+{
+  "oncogenicitySubmission": [
+    {
+      "recordStatus": "novel",
+      "localID": "varcat.variant:5",
+      "localKey": "varcat.overall_assessment:3",
+      "oncogenicityClassification": {
+        "oncogenicityClassificationDescription": "Oncogenic",
+        "dateLastEvaluated": "2024-04-01",
+        "comment": "This variant is classified oncogenic in Schwannoma. The total score for this classification is 3 and the evidence supporting this classification are as follows. Functional studies support an oncogenic effect for this variant (ClinGen/CGC/VICC 2022: OS2). This variant has not been identified in control populations (or at an extremely low minor allele frequency) within gnomAD, a large-scale population database (ClinGen/CGC/VICC 2022: OP4)."
```

Comment:
@wesleygoar See [this](https://github.com/ga4gh/gks-clinvar/issues/5#issuecomment-2101683629) comment

Reply:
We also should figure out how to show that I modulated the code on this instance. OS2 is usually strong (+4), but I modulated it to be moderate (+2) because there was some evidence supporting oncogenicity but it didn't fit the criteria for being a full +4.

Reply:
I've been manually creating these, so if you have format you'd like to follow we can add it and add support in an endpoint later.